data/data/aws/vpc/sg-worker.tf

```diff
@@ -146,6 +146,26 @@ resource "aws_security_group_rule" "worker_ingress_kubelet_insecure_from_master"
   to_port   = 10250
 }
 
+resource "aws_security_group_rule" "worker_ingress_crio_metrics" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.worker.id}"
+
+  protocol  = "tcp"
+  from_port = 9537
+  to_port   = 9537
```

Comment:
Instead of poking another single-port hole for everyone->workers, I think we want to expand [our current 9100 hole][1] to cover 9000-9999 (like we did for everyone->masters in #683 with the fixup from #1202).  That way the next metric to come along doesn't need to poke new holes, and we're more consistent between master and worker handling.

[1]: https://github.com/openshift/installer/blob/v0.12.0/data/data/aws/vpc/sg-worker.tf#L109-L127

Reply:
Fixed, but unsure about naming, please let me know if it's correct or needs changing

Reply:
> Fixed, but unsure about naming...

`worker_ingress_internal*` matches the pattern from #683, so naming looks good to me.